lib/happo/public/happo-runner.js

```diff
@@ -162,6 +162,14 @@ window.happo = {
     box.right = Math.max(box.right, rect.right);
     box.top = Math.min(box.top, rect.top);
 
+    if (box.left < 0) {
+      box.left = 0; // Don't go outside window
+    }
+
+    if (box.top < 0) {
+      box.top = 0; // Don't go outside window
+    }
```

Comment:
Am I understanding this correctly that without this, the Happo run errors out because it can't crop the screenshot to something off canvas?